### ISSUE TITLE
Email already exists!

### ISSUE THREAD

**Issue by gargbhawna** [shivank1404/Login-Signup-using-Firebase#1]
You haven't put any code snippet for checking whether the entered email already exists

**Comment by shivank1404** [comment]
I guess thats the inbuilt feature of the firebase login system but you can check the following or add the implementation.

**Comment by gargbhawna** [comment]
Yeah it's inbuilt but I want to print a particular message for that
particular authentication failure

**Comment by nagaarjunvpr** [comment]
hello how to add my own firebase
